fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -119,15 +116,15 @@ def fit(self, X, y, *, sensitive_features, **kwargs):
         threshold_optimization_method = None
         if self.constraints == DEMOGRAPHIC_PARITY:
             threshold_optimization_method = \
-                _threshold_optimization_demographic_parity
+                self._threshold_optimization_demographic_parity
         elif self.constraints == EQUALIZED_ODDS:
             threshold_optimization_method = \
-                _threshold_optimization_equalized_odds
+                self._threshold_optimization_equalized_odds
         else:
             raise ValueError(NOT_SUPPORTED_CONSTRAINTS_ERROR_MESSAGE)
 
         self._post_processed_predictor_by_sensitive_feature = threshold_optimization_method(
```

Comment:
is this intended to be `postprocessed_predictor_by_sensitive_feature_`?

Reply:
update: to expedite things, i'm fine leaving this as it is currently. we can expose this as a fitted attribute in future.

Reply:
Indeed. That will be part of my follow-up PR then. Thank you!